build/Version.props

```diff
@@ -17,6 +17,9 @@
     <NugetVersion Condition=" '$(NugetVersion)' == '' ">$(VersionMajor).$(VersionMinor).$(VersionPatch)</NugetVersion> 
 
     <SdkVersion>$(NugetVersion)</SdkVersion>
-    <SdkNugetVersion>$(NugetVersion)</SdkNugetVersion> 
+    <SdkDebianPackageVersion>$(SdkVersion)</SdkDebianPackageVersion>
+    <SdkNugetVersion>$(NugetVersion)</SdkNugetVersion>
+
+    <DotnetDebToolVersion Condition=" '$(DotnetDebToolVersion)' == '' ">1.0.0-*</DotnetDebToolVersion>
```

Comment:
This property doesn't belong here.  IMO it really belongs in the DEB specific file.  But if we feel all properties need to be removed from .targets files, then this better belongs in `DependencyVersions.props`